docs/sources/flow/getting-started/migrating-from-prometheus.md

```diff
@@ -0,0 +1,187 @@
+	targets = [{
+		__address__ = "localhost:12345",
+	}]
+	forward_to     = [prometheus.remote_write.default.receiver]
+	job_name       = "prometheus"
+	scrape_timeout = "45s"
+}
+
+prometheus.remote_write "default" {
+	endpoint {
+		name = "grafana-cloud"
+		url  = "https://prometheus-us-central1.grafana.net/api/prom/push"
+
+		basic_auth {
+			username = "USERNAME"
+			password = "PASSWORD"
+		}
+
+		queue_config {
+			capacity             = 2500
+			max_shards           = 200
+			max_samples_per_send = 500
+		}
+
+		metadata_config {
+			max_samples_per_send = 500
+		}
+	}
+}
+```
+
+#### Debugging
+
+If Prometheus configuration is provided that cannot be converted,
+diagnostic information is printed to `stderr`. You can bypass
+any non-critical issues and output the flow configuration using best
+effort conversion by including the `--bypass-errors` flag.
+   
+> Be aware that the behavior may not match when bypassing errors.
+
+```bash
+grafana-agent convert --format=prometheus --bypass-errors --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+```
+
+You can also output a diagnostic report by including the `--report` flag.
+
+```bash
+grafana-agent convert --format=prometheus --report=OUTPUT_REPORT_PATH --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+```
+
+> Replace `OUTPUT_REPORT_PATH` with the full path to output the report to.
+
+Using the example Prometheus configuration from above outputs a diagnostic
+report:
+
+```
+(Info) Converted scrape_configs job_name "prometheus" into...
+  A prometheus.scrape.prometheus component
+(Info) Converted 1 remote_write[s] "grafana-cloud" into...
+  A prometheus.remote_write.default component
+```
+
+## Run a Prometheus configuration
+
+If you’re not ready to switch over to flow configuration, you can also run
+the Prometheus configuration without having to save it as a flow config.
+This allows you to try flow mode without having to modify your existing
+Prometheus configuration infrastructure.
+
+> In this task, we will use the [run][] CLI command to output flow
+> configuration from a Prometheus configuration.
+
+1. Execute the following to start the Grafana Agent in flow mode with a
+Prometheus configuration:
+
+```bash
+grafana-agent run --config.format=prometheus INPUT_CONFIG_PATH
+```
+
+> Replace `INPUT_CONFIG_PATH` with the full path to the Prometheus configuration.
+
+#### Debugging
+
+If Prometheus configuration is provided that cannot be converted,
+diagnostic information is printed to `stderr`. You can bypass
+any non-critical issues and start the Agent by including the
+`--config.bypass-conversion-errors` flag.
+
+> Be aware that the behavior may not match when bypassing errors
+> and doing so should be avoided in Production systems.
+
+```bash
+grafana-agent run --config.format=prometheus --config.bypass-conversion-errors INPUT_CONFIG_PATH
+```
+
+If a deeper dive is needed for Prometheus configuration conversion issues,
+use the [convert][] CLI command [debugging][]
```

Comment:
I'm a little confused by this section. What does "use the convert CLI command debugging" mean? If this is trying to link to a section, I think it can be more clear about that rather than trying to interweave the links into the sentence naturally.

Reply:
new organization and words have been added for this section